integ/components/deadline/deadline_03_workerFleetHttp/test/deadline_03_workerFleetHttp.test.ts

```diff
@@ -123,7 +123,8 @@ describe.each(testCases)('Deadline WorkerFleet tests (%s)', (_, id) => {
       };
       return awaitSsmCommand(bastionId, params).then( response => {
         var responseOutput = response.output;
-        expect(responseOutput).toMatch(/testpool\ntestgroup\ntestregion/);
+        // We are not expecting testregion because Deadline worker return unrecognized if region does not exist
+        expect(responseOutput).toMatch(/testpool\ntestgroup\nunrecognized/);
```

Comment:
Where do we set these values "testgroup", "testpool" and "testregion" ?
Maybe we could add a test with region = "us-west-2" just to be sure, that we get the correct result if the region exists, because now we only test for "unrecognized".

Reply:
By default we have only `unrecognized` region. So we can only test that we have `unrecognized` instead of `none` by default.

Reply:
Is it possible for us to add to the test to have it create the region.

Reply:
also this change makes it so these tests no longer work with older versions of Deadline. Is the plan to drop support for those versions of Deadline?

Reply:
Code was modified to expect both cases with `urecognized` region and `testregion`

Reply:
This looks good to me. Resolving the conversation but tagging @grbartel so he notices the change.